airflow/api_connexion/openapi/v1.yaml

```diff
@@ -1112,7 +1112,7 @@ paths:
 
   /version:
     get:
-      summary: Get a version information
+      summary: Get Airflow version information
```

Comment:
```suggestion
      summary: Get Airflow version
```
Ideally, this field should be less than 20 characters long.

![Screenshot 2020-08-25 at 01 36 00](https://user-images.githubusercontent.com/12058428/91106598-5af73780-e673-11ea-9c07-fdf767158f1c.png)

Reply:
fixed